src/python/espressomd/constraints.py

```diff
@@ -117,17 +117,17 @@ class ShapeBasedConstraint(Constraint):
     Examples
     ----------
     >>> import espressomd
-    >>> from espressomd import shapes
-    >>> system = espressomd.System()
+    >>> import espressomd.shapes
+    >>> system = espressomd.System(box_l=[1., 1., 1.])
```

Comment:
good catch, in fact this code example doesn't even work because you can't have a particle at exactly the center of a sphere (the surface vector has no orientation since the particle is equidistant to all points on the sphere surface)